README.markdown

```diff
@@ -31,7 +31,7 @@
 
 ## Installation
 
-First you'll need to make sure your system has a c++ compiler. For OS X, Xcode will work, for Ubuntu, the `build-essential` and `libssl-dev` packages work.
+If you're going to install nodejs or io.js from its source code, you'll need to make sure your system has a c++ compiler. For OS X, Xcode will work, for Debian/Ubuntu based GNU/Linux, the `build-essential` and `libssl-dev` packages work.
```

Comment:
We should specify here that "installing from its source code" is only necessary when there is not a prepackaged binary available for your system.